packages/svelte/src/compiler/phases/3-transform/css/index.js

```diff
@@ -198,7 +198,7 @@ const visitors = {
 	SelectorList(node, { state, next, path }) {
 		// Only add comments if we're not inside a complex selector that itself is unused or a global block
 		if (
-			(!is_in_global_block(path) || node.children.length > 1) &&
+			(!is_in_global_block(path) || (node.children.length > 1 && is_in_global_block(path, true))) &&
```

Comment:
couldn't this just be `path.at(-1).metadata.is_global_block` instead of changing the `is_in_global_block` signature?

Reply:
I throught we can't do that since `SelectorList` can also be the child of a `PseudoClassSelector` but in that case we shouldn't bail on children greated 1, too, so this fix is actually incomplete; will adjust 👍